data/helpers.d/utils

```diff
@@ -24,3 +24,28 @@ ynh_get_plain_key() {
         fi
     done
 }
+
+# Curl abstraction to help with POST requests to local pages (such as installation forms)
+#
+# example: ynh_local_curl "/install.php?installButton" "foo=$var1" "bar=$var2"
+# 
+# usage: ynh_local_curl "page_uri" "key1=value1" "key2=value2" ...
+# | arg: page_uri    - Path (relative to $path_url) of the page where POST data will be sent
+# | arg: key1=value1 - (Optionnal) POST key and corresponding value
+# | arg: key2=value2 - (Optionnal) Another POST key and corresponding value
+# | arg: ...         - (Optionnal) More POST keys and values
+ynh_local_curl () {
+	# Define url of page to curl
+	full_page_url=https://localhost$path_url$1
+
+	# Concatenate all other arguments with '&' to prepare POST data
+	POST_data=""
+	for arg in "${@:2}"
+	do
+		POST_data="${POST_data}${arg}&"
+	done
+	POST_data=${POST_data::-1}
```

Comment:
I guess this line remove the last '&'.
Please comment this kind of line.

I think is really not clear for everyone that don't know bash very well.